### PR TITLE
Transformed listview into recycler view. Added event listener to brow…

### PR DESCRIPTION
**Pull request by silvioguiso**
…se to the documentation of the language.

### CODE REVIEW

File: app/src/main/java/edu/cascadia/mobile/apps/arraylistplay/StringArrayAdapter.java [hunk 1]
```diff
@@ -1,52 +1,98 @@
...
+                        break;
+                    case "Swift":
+                        browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://swift.org/documentation/"));
+                        break;
+                    default:
+                        break;
+                }
+
+                v.getContext().startActivity(browserIntent);
+            }
```
Comment (Bansenauer-Cascadia) [inline app/src/main/java/edu/cascadia/mobile/apps/arraylistplay/StringArrayAdapter.java:59]: Good work. 
Your event handling works well, but violates the Single Responsibility principle.
Think about how you might move this code out of the Adapter and back into the MainActivity. 
Take a look at the Chapter 13 code for an example

**Review by Bansenauer-Cascadia — COMMENTED** [review]
Good work - see my comment in the code

**Comment by Bansenauer-Cascadia** [comment]
Nice work.

File: app/src/main/java/edu/cascadia/mobile/apps/arraylistplay/StringArrayAdapter.java (hunk 1 above)
Comment (silvioguiso) [reply app/src/main/java/edu/cascadia/mobile/apps/arraylistplay/StringArrayAdapter.java:59]: Awesome, thanks for the feedback! I just added that and tested the implementation. Commit pushed @ https://github.com/silvioguiso/HFA-Chapter07/commit/1f96c3a84437fd375d3e83e3a63929f3a2b1bd52

**Pull request closed without merging**